src/common/storage/storage-manager.js

```diff
@@ -64,6 +65,19 @@ export default class StorageManager {
       StorageWrapper.setItem(StorageManager.StorageKeys.TEXT_LANG, textTrack.language);
     });
 
+    eventManager.listen(player, player.Event.UI.UI_ELEMENT_CLICKED, event => {
```

Comment:
why do we need the event ?

Reply:
to save only when the language is selected by the user not by app config for example